recipes/apoc/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "apoc" %}
```

Ok so I spoke with the team and basically we need to consider `bioconda` and `conda-forge` a single unified namespace. That means we can't use `apoc` here. I suggest using the following, but feel free to suggest something else, @haesleinhuepf!

```suggestion
{% set name = "apoc-backend" %}
```

---

Hi @jaimergp , this is just the conda package name, right? I don't need to change anything in the python code, correct? If so, I'm fine with that suggestion. 🙂

---

Yep, it's only what `conda` sees. The import name itself should be ok.